lib/report/report.go

```diff
@@ -102,9 +103,18 @@ type Config struct {
 	// Packages specifies the package service for the package
 	// diagnostics collector
 	Packages pack.PackageService
+	// Since specifies the start of the time filter. A value of 1h will report
+	// log entries starting from one ago up till the end of the time filter.
+	Since time.Duration
+	// Until specifies the end of the time filter. A value of 1h will report
+	// log entries from the start of the time filter up till one hour ago.
+	Until time.Duration
```

Comment:
Maybe clarify it's not being used currently and add a TODO item, or just remove completely for now so we don't keep dead code.